conf/alluxio-env.sh.template

```diff
@@ -1,110 +1,39 @@
-else
-  # Assuming Linux
-  if [[ -d /usr/lib/jvm/java-7-oracle ]]; then
-    export JAVA_HOME=${JAVA_HOME:-"/usr/lib/jvm/java-7-oracle"}
-  else
-    # openjdk will set this
-    if [[ -d /usr/lib/jvm/jre-1.7.0 ]]; then
-      export JAVA_HOME=${JAVA_HOME:-"/usr/lib/jvm/jre-1.7.0"}
-    fi
-  fi
-  export ALLUXIO_RAM_FOLDER=${ALLUXIO_RAM_FOLDER:-"/mnt/ramdisk"}
-fi
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
 
-export JAVA_HOME=${JAVA_HOME:-"$(dirname $(which java))/.."}
-
-export JAVA="${JAVA_HOME}/bin/java"
-export ALLUXIO_MASTER_ADDRESS=${ALLUXIO_MASTER_ADDRESS:-XXX_MASTER_ADDRESS_XXX}
-export ALLUXIO_UNDERFS_ADDRESS=${ALLUXIO_UNDERFS_ADDRESS:-XXX_UNDER_STORAGE_XXX}
-export ALLUXIO_WORKER_MEMORY_SIZE=${ALLUXIO_WORKER_MEMORY_SIZE:-XXX_MEMORY_SIZE_XXX}
-export ALLUXIO_SSH_FOREGROUND=${ALLUXIO_SSH_FOREGROUND:-"yes"}
-export ALLUXIO_WORKER_SLEEP=${ALLUXIO_WORKER_SLEEP:-"0.02"}
+# This file contains the environment variables required to run Alluxio. Copy it as alluxio-env.sh and
+# edit that to configure Alluxio for your site. At a minimum,
+# the following variables should be set:
 
-# Prepend Alluxio classes before classes specified by ALLUXIO_CLASSPATH
-# in the Java classpath.  May be necessary if there are jar conflicts
-#export ALLUXIO_PREPEND_ALLUXIO_CLASSES=${ALLUXIO_PREPEND_ALLUXIO_CLASSES:-"yes"}
+# Where the Alluxio deployment is installed
+# ALLUXIO_HOME
 
 # Where log files are stored. $ALLUXIO_HOME/logs by default.
-#export ALLUXIO_LOGS_DIR=${ALLUXIO_LOGS_DIR:-${ALLUXIO_HOME}/logs}
+# ALLUXIO_LOGS_DIR
 
-XXX_ADDITIONAL_SETTINGS_XXX
+# Address of the master
+# ALLUXIO_MASTER_ADDRESS
 
-CONF_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# Where worker stores in memory data. Consider setting it to /dev/shm if tmpFS is used.
+# ALLUXIO_RAM_FOLDER
 
-export ALLUXIO_JAVA_OPTS+="
-  -Dlog4j.configuration=file:${CONF_DIR}/log4j.properties
-  -Dalluxio.worker.tieredstore.levels=1
-  -Dalluxio.worker.tieredstore.level0.alias=MEM
-  -Dalluxio.worker.tieredstore.level0.dirs.path=${ALLUXIO_RAM_FOLDER}
-  -Dalluxio.worker.tieredstore.level0.dirs.quota=${ALLUXIO_WORKER_MEMORY_SIZE}
-  -Dalluxio.underfs.address=${ALLUXIO_UNDERFS_ADDRESS}
-  -Dalluxio.worker.memory.size=${ALLUXIO_WORKER_MEMORY_SIZE}
-  -Dalluxio.master.hostname=${ALLUXIO_MASTER_ADDRESS}
-  -Dorg.apache.jasper.compiler.disablejsr199=true
-  -Djava.net.preferIPv4Stack=true
-"
+# Address of the under filesystem address.
+# ALLUXIO_UNDERFS_ADDRESS
 
-# Master specific parameters. Default to ALLUXIO_JAVA_OPTS.
-export ALLUXIO_MASTER_JAVA_OPTS="${ALLUXIO_JAVA_OPTS}"
+# How much memory to use (e.g. 1000mb, 2gb) per worker
+# ALLUXIO_WORKER_MEMORY_SIZE
 
-# Worker specific parameters that will be shared to all workers. Default to ALLUXIO_JAVA_OPTS.
-export ALLUXIO_WORKER_JAVA_OPTS="${ALLUXIO_JAVA_OPTS}"
+ALLUXIO_MASTER_ADDRESS=${ALLUXIO_MASTER_ADDRESS:-XXX_MASTER_ADDRESS_XXX}
```

Comment:
we should get rid of all of these, otherwise it will not be possible to override these from the properties file ... instead of sed-ing this file, the bootstrap script should instead generate alluxio.properties file